utils/ApiError.js

```diff
@@ -0,0 +1,14 @@
+class ApiError extends Error {
+    constructor(statusCode, message, isOperational = true, stack = "") {
+      super(message);
+      this.statusCode = statusCode;
+      this.isOperational = isOperational;
```

Comment:
Codacy has a fix for the issue: [Replace `······` with `····`](https://app.codacy.com/gh/ParmenidesSartre/Respond-Task1/pullRequest?prid=9934277)
```suggestion
    this.isOperational = isOperational;
```